### PR TITLE
Extend length of URL path in logged request

### PR DESCRIPTION
**Pull request by rubickcz**
This PR extends length of `path` field in logged request.

### CODE REVIEW

File: security/models.py [hunk 1]
```diff
@@ -162,7 +162,7 @@ class LoggedRequest(SmartModel):
     host = models.CharField(_('host'), max_length=255, null=False, blank=False, db_index=True)
     host._filter = CaseSensitiveStringFieldFilter
     method = models.SlugField(_('method'), max_length=7, null=False, blank=False, db_index=True)
-    path = models.CharField(_('URL path'), max_length=255, null=False, blank=True, db_index=True)
+    path = models.CharField(_('URL path'), max_length=2000, null=False, blank=True, db_index=True)
```
Comment (rubickcz) [inline security/models.py:165]: A valid URL can be more than 255 chars long. I used 2000 which should be enough and we're using this value across projects for URL lengths.

**Comment by matllubos** [comment]
please do rebase

**Comment by rubickcz** [comment]
@matllubos I cannot rebase - this is branched off an older version - we cannot merge it now. See comment in bitbucket.

**Comment by rubickcz** [comment]
@matllubos rebased.

**Pull request merged**